.github/workflows/stale.yml

```
@@ -0,0 +1,39 @@
+# This workflow warns of PRs that have had no activity for a specified amount of time.
+#
+# For more information, see https://github.com/actions/stale
+name: Mark stale pull requests
+
+on:
+  # Run every day at 00:00 UTC
+  schedule:
+    - cron: '0 0 * * *'
+  # Or run on demand
+  workflow_dispatch:
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+
+    steps:
+      - uses: actions/stale@v9
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+          days-before-pr-stale: 14
+          days-before-issue-stale: -1   # we don't use issues
+          days-before-close: -1         # don't close stale PRs/issues
+          exempt-draft-pr: true         # don't mark draft PRs as stale
+          stale-pr-label: "stale"       # label to use when marking as stale
+
+          stale-pr-message: >
+            This PR has not had activity in the past 2 weeks, labeling it as stale.
+            Any new activity will remove the stale label. To attract more reviewers, tag
```

And wondering about pull requests with merge conflicts e.g. to suggest or not to suggest to try and resolve any merge conflicts. Maybe that would be a different kind of bot though.
```suggestion
            Any new activity will remove the stale label. To attract more reviewers, please tag
```

---

I don't think it matters whether there are merge conflicts or not. If a PR is stale, it should be quite obvious what next steps are? Or is it unclear? A contributor should aim to get all lights green, and then ask for review. But if the contributor needs help she can of course ask for help at any time.